stories/TypeScriptComponent.tsx

```diff
@@ -40,6 +40,7 @@ type TypeScriptComponentProps = {
   'quoted-prop': any;
   // @internal
   hideProp: Boolean;
+  nocomment: boolean;
```

Comment:
👀

Reply:
Hah yeah that was intentional to test what happens when a "description" isn't provided :)